package.json

```
@@ -14,8 +14,8 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "4.2.2",
-    "bootstrap": "^4.5.3",
-    "jquery": "^3.5.1",
+    "bootstrap": "^5.0.0",
+    "jquery": "^3.6.0",
```

バージョンは変更しない方がよいでしょう。

```suggestion
    "bootstrap": "^4.5.3",
    "jquery": "^3.5.1",
```